delivery_price_method/models/delivery_carrier.py

```diff
@@ -13,7 +13,6 @@ class DeliveryCarrier(models.Model):
             ("fixed", "Fixed price"),
             ("base_on_rule", "Based on Rules"),
         ],
-        string="Price method",
```

Comment:
This is changing the string to translate. Please put the pylint ignore to avoid the error in linter. This has been fixed in later pylint-odoo version